modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFClaim.testcase

```diff
@@ -93,75 +101,141 @@ definition {
+				listLeads = "Document_1.xlsx",
+				reimbursementInvoice = "Document_1.png",
+				thirdInvoice = "Document_1.png");
+		}
+
+		task ("When Login with the Manager user and go to Partner Portal") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "manager@liferay.com",
+				userLoginFullName = "manager");
+
+			PRMNavigator.openSitePage(pageName = "MDF Requests");
+		}
+
+		task ("And I go to MDF claim listing page") {
+			PRMNavigator.goToNavItem(navItem = "MDF Claim");
+		}
 
-			Button.click(button = "Continue");
+		task ("And I export all visible Claims") {
+			Button.click(button = "Export MDF Claim");
+		}
 
-			Button.click(button = "Continue");
+		task ("Then the information about the visible Claims will be inside the exported file") {
+			var firstID = PRMMDFClaim.getClaimID(claimIndex = 1);
+			var secondID = PRMMDFClaim.getClaimID(claimIndex = 2);
 
-			Button.click(button = "Submit");
+			PRMMDFRequest.viewExportedMDF(
+				fileContentList = "${firstID},${secondID}",
+				fileName = "MDF Claim.csv");
 		}
+	}
+
+	@description = "This is a test for LPS-178119. Verify that is possible to export all visible Claims"
+	@priority = 4
+	test CanExportFilteredClaims {
+		task ("Given a new account is created") {
+			JSONAccountEntry.addAccountEntry(
+				accountEntryName = "Company Name 2",
+				accountEntryType = "Business");
 
-		task ("And given that the manager user is logged in and goes to MDF listing page") {
+			JSONAccountEntryUser.addExistUsertoAccount(
+				accountEntryName = "Company Name 2",
+				userEmailAddress = "partner@liferay.com");
+
+			JSONAccountEntryUser.addExistUsertoAccount(
+				accountEntryName = "Company Name 2",
+				userEmailAddress = "manager@liferay.com");
+		}
+
+		task ("And a new MDF is created with a different account") {
+			JSONPRM.createMDFRequest(
+				accountEntryName = "Company Name 2",
+				activityName = "Activity Name 3",
+				campaignName = "Campaign Name 1",
+				secondActivityName = "Activity Name 4",
+				secondBudgetValueList = 1000);
+		}
+
+		task ("And given that the Manager user is logged in and goes to MDF listing page") {
 			User.logoutAndLoginPG(
 				userLoginEmailAddress = "manager@liferay.com",
 				userLoginFullName = "manager");
 
-			PRMUtils.gotoPRM();
+			PRMNavigator.openSitePage(pageName = "MDF Requests");
+		}
+
+		task ("And the manager approves the MDF's") {
+			PRMMDFRequest.goToMDF(campaignName = "Campaign Name");
+
+			PRMMDFRequest.changeMDFStatus(key_statusButton = "Approve");
+
+			PRMNavigator.openSitePage(pageName = "MDF Requests");
 
-			PRMHomePage.gotoQuickLinksOption(
-				buttonLink = "mdf-requests",
-				buttonText = "MDF");
+			PRMMDFRequest.goToMDF(campaignName = "Campaign Name 1");
+
+			PRMMDFRequest.changeMDFStatus(key_statusButton = "Approve");
 		}
 
-		task ("And the MDF is approved") {
-			PRMMDFRequest.goToMDF(campaignName = "Campaign Name 2");
+		task ("When Login with the Partner user and go to Partner Portal") {
```

Comment:
Hi @marinaasant0s, it's a little confuse. I think it should be 'And the partner logs in...'